app/scripts/modules/google/src/loadBalancer/configure/http/commandBuilder.service.js

```diff
@@ -136,7 +136,7 @@ module.exports = angular.module('spinnaker.deck.gce.httpLoadBalancer.backing.ser
     }
 
     function getHealthChecks () {
-      return gceHealthCheckReader.listHealthChecks('HTTP');
+      return gceHealthCheckReader.listHealthChecks();
```

Comment:
Would we still need the `listHealthChecks` that takes a string argument?

Reply:
Good question - looks like autohealing policies require (or maybe required) HTTP health checks. I'll check if they still do.